packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.util.ts

```diff
@@ -207,6 +209,10 @@ export const getBuildType = (
     return 'test';
   }
 
+  if (webex.internal.caBuildType) {
+    return webex.internal.caBuildType;
```

Comment:
it seems ok to have an escape hatch, though perhaps it would be better to have it as part of the config for metrics, rather than just a flag applied to webex.internal? 
e.g.
```
webex.metrics.config.caBuildType
```